src/platforms/ios/iosutils.mm

```diff
@@ -113,3 +113,12 @@
       return -1;
   }
 }
+
+// static
+void IOSUtils::setStatusBarTextColor(quint8 color) {
+  if (color == 0) {
+    [UIApplication sharedApplication].statusBarStyle = UIStatusBarStyleLightContent;
+  } else {
+    [UIApplication sharedApplication].statusBarStyle = UIStatusBarStyleDarkContent;
```

Comment:
This is the deprecated APIs I mentioned on the PR description. They were deprecated on iOs 13.